src/components/Request/FormFields/LinkedinSearchComments.tsx

```diff
@@ -0,0 +1,43 @@
+import { Label, Text, Textarea } from "@fluentui/react-components";
+import { Controller } from "react-hook-form";
+import { FormField } from "components/Request/NewRequestForm";
+import "components/Request/Request.css";
+import { TextFieldIcon } from "@fluentui/react-icons-mdl2";
+
+const LinkedinSearchComments = ({ name, form }: FormField) => {
+  return (
+    <div className="requestFieldContainer">
+      <Label
+        htmlFor={name + "Id"}
+        weight="semibold"
+        className="requestFieldLabel"
+        required
```

Comment:
Is this required?  If so, leave it and add a rule to the input.  Otherwise drop.